src/megatron/energon/flavors/webdataset/standard_webdataset.py

```diff
@@ -0,0 +1,64 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.
+# SPDX-License-Identifier: BSD-3-Clause
+
+from typing import Generic, Type, TypeVar
+
+from megatron.energon.epathlib import EPath
+from megatron.energon.flavors.base_dataset import Sample
+from megatron.energon.flavors.webdataset.default_decoder_webdataset import (
+    DefaultDecoderWebdatasetFactory,
+)
+
+T_sample = TypeVar("T_sample", covariant=True)
+
+
+class StandardWebdatasetFactory(DefaultDecoderWebdatasetFactory[T_sample], Generic[T_sample]):
+    """
+    Basic class for any dataset which contains images and / or videos. Applies default wds loading logic for all
```

Comment:
Docstring outdated

Reply:
Huh, why that? What's outdated here?

Reply:
This is the same docstring as for `DefaultDecoderWebdatasetFactory`. And also: Why would this not work for text-only?